ts/features/wallet/cobadge/component/CobadgeCard.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import {
+  Image,
+  ImageSourcePropType,
+  ImageStyle,
+  StyleProp,
+  StyleSheet
+} from "react-native";
+import { View } from "native-base";
+import I18n from "../../../../i18n";
+import BaseCardComponent from "../../component/BaseCardComponent";
+import { useImageResize } from "../../onboarding/bancomat/screens/hooks/useImageResize";
+import { H4 } from "../../../../components/core/typography/H4";
+import { H5 } from "../../../../components/core/typography/H5";
+import abiLogoFallback from "../../../../../img/wallet/cards-icons/abiLogoFallback.png";
+
+type Props = {
+  caption?: string;
+  expireMonth?: string;
+  expireYear?: string;
+  abiLogo?: string;
+  brandLogo: ImageSourcePropType;
+};
+
+const styles = StyleSheet.create({
+  abiLogoFallback: {
+    width: 40,
+    height: 40,
+    resizeMode: "contain"
+  },
+  brandLogo: {
+    width: 80,
+    height: 40,
+    resizeMode: "contain"
+  },
+  bankName: { textTransform: "capitalize" }
+});
+
+const BASE_IMG_W = 160;
+const BASE_IMG_H = 40;
+
+const CobadgeCard: React.FunctionComponent<Props> = (props: Props) => {
+  const imgDimensions = useImageResize(BASE_IMG_W, BASE_IMG_H, props.abiLogo);
+
+  const imageStyle: StyleProp<ImageStyle> | undefined = imgDimensions.fold(
+    undefined,
+    imgDim => ({
+      width: imgDim[0],
+      height: imgDim[1],
+      resizeMode: "contain"
+    })
+  );
+  return (
+    <BaseCardComponent
+      topLeftCorner={
+        <>
+          {props.abiLogo && imageStyle ? (
+            <Image
+              source={{ uri: props.abiLogo }}
+              style={imageStyle}
+              testID={"abiLogo"}
+            />
+          ) : (
+            <Image
+              source={abiLogoFallback}
+              style={styles.abiLogoFallback}
+              testID={"abiLogoFallback"}
+            />
```

Comment:
On iOS I always see the fallback image. 


https://user-images.githubusercontent.com/26501317/106308043-70930d80-6260-11eb-8e70-bb2f54612a7c.mov

```suggestion
          {props.abiLogo && imageStyle ? (
            <Image
              source={{ uri: props.abiLogo }}
              style={imageStyle}
              testID={"abiLogo"}
              key={"abiLogo"}
            />
          ) : (
            <Image
              source={abiLogoFallback}
              style={styles.abiLogoFallback}
              testID={"abiLogoFallback"}
              key={"abiLogoFallback"}
            />
```
My guess is that the node occupies the same position and the cache is not refreshered, obtaining something similar to https://github.com/facebook/react-native/issues/9195. 
Using different keys it renders correctly for me both on iOS and Android.

Reply:
Thanks for the suggestion. Working always on the same card I never notice this bug.
Fixed in [a6c4e84](https://github.com/pagopa/io-app/pull/2751/commits/a6c4e84ab2755f11f24c1a0262a820fc877be78c)